wondershaper: add page (#8034)
tldr-pages/tldr: pages/common/wondershaper.md

# wondershaper

> > Wonder Shaper is a script that allows the user to limit the bandwidth of one or more network adapters.

Reviewer: ```suggestion
> Allows the user to limit the bandwidth of one or more network adapters.
```

> More information: <https://github.com/magnific0/wondershaper#usage>.

> - Print the [h]elp:

Reviewer: ```suggestion
- Display help:
```

> - Display help:
> 
> `wondershaper -h`

Reviewer 2: ```suggestion
- Print the [h]elp:

`wondershaper -h`
```

Reviewer 2: It's better to highlight the first character of the short option in the command description.

> `wondershaper -h`

Reviewer: Does this have long versions of these flags such as --help instead?

Author: no, it doesn't have

> - Show the current status of adapter:

Reviewer 2: ```suggestion
- Show the current [s]tatus of [a]dapter:
```

> - Show the current [s]tatus of [a]dapter:

Reviewer: ```suggestion
- Show the current [s]tatus of an [a]dapter:
```

Reviewer 2: Maybe better:

```suggestion
- Show the current [s]tatus of a specific [a]dapter:
```

`wondershaper -s -a {{adapter_name}}`

> - Clear the limits from adapter:

Reviewer 2: ```suggestion
- Clear the limits from [a]dapter:
```

> - Clear the limits from [a]dapter:

Reviewer: ```suggestion
- Clear the limits from an [a]dapter:
```

Reviewer 2: The same thing here:

```suggestion
- Clear limits from a specific [a]dapter:
```

`wondershaper -c -a {{adapter_name}}`

> - Set maximum download rate (in Kbps):

Reviewer 2: ```suggestion
- Set maximum [d]ownload rate (in Kbps):
```

> - Set maximum [d]ownload rate (in Kbps):

Reviewer: ```suggestion
- Set a maximum [d]ownload rate (in Kbps):
```

Reviewer 2: ```suggestion
- Set a specific maximum [d]ownload rate (in Kbps):
```

> `wondershaper -a {{adapter_name}} -d 1024`

Reviewer: ```suggestion
`wondershaper -a {{adapter_name}} -d {{1024}}`
```

> - Set maximum upload rate (in Kbps):

Reviewer 2: ```suggestion
- Set maximum [u]pload rate (in Kbps):
```

> - Set maximum [u]pload rate (in Kbps):

Reviewer: ```suggestion
- Set a maximum [u]pload rate (in Kbps):
```

Reviewer 2: ```suggestion
- Set a specific maximum [u]pload rate (in Kbps):
```

> `wondershaper -a {{adapter_name}} -u 512`

Reviewer: ```suggestion
`wondershaper -a {{adapter_name}} -u {{512}}`
```

> - Set both maximum download rate and upload rate (in Kpbs):

Reviewer 2: ```suggestion
- Set both maximum [d]ownload rate and [u]pload rate (in Kpbs):
```

> - Set both maximum [d]ownload rate and [u]pload rate (in Kpbs):

Reviewer: ```suggestion
- Set both a maximum [d]ownload rate and a [u]pload rate (in Kpbs):
```

Reviewer 2: ```suggestion
- Set a specific maximum [d]ownload rate and [u]pload rate (in Kpbs):
```

I am not sure just about `a` here: is it right to use it when we are talking about several things?

> `wondershaper -a {{adapter_name}} -d 1024 -u 512`

Reviewer: ```suggestion
`wondershaper -a {{adapter_name}} -d {{1024}} -u {{512}}`
```
